cf-conventions.adoc

```diff
@@ -1,13 +1,36 @@
 include::version.adoc[]
 = NetCDF Climate and Forecast (CF) Metadata Conventions
-Brian{nbsp}Eaton; Jonathan{nbsp}Gregory; Bob{nbsp}Drach; Karl{nbsp}Taylor; Steve{nbsp}Hankin; Jon{nbsp}Blower; John{nbsp}Caron; Rich{nbsp}Signell; Phil{nbsp}Bentley; Greg{nbsp}Rappa; Heinke{nbsp}Höck; Alison{nbsp}Pamment; Martin{nbsp}Juckes; Martin{nbsp}Raspaud; Randy{nbsp}Horne; Timothy{nbsp}Whiteaker; David{nbsp}Blodgett; Charlie{nbsp}Zender; Daniel{nbsp}Lee; David{nbsp}Hassell; Alan{nbsp}D.{nbsp}Snow; Tobias{nbsp}Kölling; Dave{nbsp}Allured; Aleksandar{nbsp}Jelenak; Anders{nbsp}Meier{nbsp}Soerensen; Lucile{nbsp}Gaultier; Sylvain{nbsp}Herlédan; Fernando{nbsp}Manzano; Lars{nbsp}Bärring; Christopher{nbsp}Barker; Sadie{nbsp}Bartholomew
-Version{nbsp}{current-version},{nbsp}{nbsp}{docprodtime}: See{nbsp}https://cfconventions.org{nbsp}for{nbsp}further{nbsp}information.
+Brian Eaton; Jonathan Gregory; Bob Drach; Karl Taylor; Steve Hankin; Jon Blower; John Caron; Rich Signell; Phil Bentley; Greg Rappa; Heinke Höck; Alison Pamment; Martin Juckes; Martin Raspaud; Randy Horne; Timothy Whiteaker; David Blodgett; Charlie Zender; Daniel Lee; David Hassell; Alan D. Snow; Tobias Kölling; Dave Allured; Aleksandar Jelenak; Anders Meier Soerensen; Lucile Gaultier; Sylvain Herlédan; Fernando Manzano; Lars Bärring; Christopher Barker; Sadie Bartholomew
```

Comment:
I think the non-breaking spaces were there to avoid having a linebreak between the first name and the family name. Are they not needed with the new formatting you are working on?